nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-resources/src/main/resources/conf/nifi.properties

```diff
@@ -345,6 +345,14 @@ nifi.diagnostics.on.shutdown.max.filecount=10
 # The diagnostics folder's maximum permitted size in bytes. If the limit is exceeded, the oldest files are deleted.
 nifi.diagnostics.on.shutdown.max.directory.size=10 MB
 
+# Performance tracking properties
+## Specifies what percentage of the time we should track the amount of time processors are using CPU, reading from/writing to content repo, etc.
+## This can be useful to understand which components are the most expensive and to understand where system bottlenecks may be occurring.
+## The value must be in the range of 0 (inclusive) to 100 (inclusive). A larger value will produce more accurate results, while a smaller value may be
+## less expensive to compute.
+## Results can be obtained by running "nifi.sh diagnostics <filename>" and then inspecting the produced file.
+nifi.performance.tracking.percentage=${nifi.performance.tracking.percentage}
```

Comment:
It would be helpful to include this same information in the Administrator's Guide, perhaps under `Runtime Monitoring Properties` or `NiFi Diagnostics`. If it is specific to the `diagnostics` command, should the property be named `nifi.diagnostics.performance.tracking.percentage`?

Reply:
Fair enough, I can update the admin guide. I don't want to mention `diagnostics` in the property name. These are not really related to diagnostics necessarily. That just happens to be the only mechanism by which we can expose them currently. But in the future, I can imagine them being exposed via Reporting Tasks and/or the UI, etc.

Reply:
Thanks for clarifying the potential uses and updating the admin guide. It seemed like there could be other ways to provide the information based on the implementation, so keeping the property name as defined sounds good.